charts/stable/emby/values.yaml

```diff
@@ -6,37 +6,41 @@
 #
 
 image:
+  # -- image repository
   repository: ghcr.io/k8s-at-home/emby
+  # -- image pull policy
   pullPolicy: IfNotPresent
-  tag: v4.5.4.0
+  # -- image tag
+  tag: v4.6.1.0
 
-strategy:
-  type: Recreate
-
-# See more environment variables in the emby documentation
-# https://emby.org/docs
-env: {}
-  # TZ:
+# -- environment variables. See more environment variables in the [emby documentation](https://emby.org/docs)
+env:
+  # @default -- See below
+  TZ: UTC
```

Comment:
```suggestion
# -- environment variables. See more environment variables in the [emby documentation](https://emby.org/docs)
# @default -- See below
env:
  # -- Set the container timezone
  TZ: UTC
```
This would require a re-rendering of the docs